src/rpc/RPC.mli

```diff
@@ -0,0 +1,2 @@
+(* Runs an RPC server that takes calls on stdin and sends results to stdout. *)
+val main : Cap.all_caps -> unit
```

Comment:
you can actually pass all_caps here, but then restrict it for the callback, so for example don't pass the stdin/stdout capability to the callback so we're sure they can't interfere.

Reply:
Yeah, right now we don't actually plumb any caps down (but we should, since we're writing to the filesystem!), but we will definitely want to restrict them once we do.